types/spdx-correct/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for spdx-correct 2.0
+// Project: https://github.com/jslicense/spdx-correct.js#readme
+// Definitions by: Jinwoo Lee <https://github.com/jinwoo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function spdxCorrect(identifier: string): string|null;
+declare namespace spdxCorrect {
```

Comment:
The namespace-merge hack may stop working in the future as node starts adding module support. Instead just use the `import x = require('x')` syntax to import.

Reply:
Oh, ok. I've been wondering how typescript will support ES6 modules in Node.

Done.